addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -481,21 +534,44 @@ odoo.define('website_slides.quiz', function (require) {
          * according to quiz result.
          *
          * @private
-         * @param OdooEvent ev
+         * @param
          */
-        _onSubmitQuiz: function (ev) {
-            var inputs = this.$('input[type=radio]:checked');
-            var values = [];
-            for (var i = 0; i < inputs.length; i++){
-                values.push(parseInt($(inputs[i]).val()));
-            }
-
-            if (values.length === this.quiz.questionsCount){
-                this._submitQuiz(values);
+        _onSubmitQuiz: function () {
```

Comment:
"on" suggests an event was triggered, but it's not the case anymore.

I think you could easily merge `_onSubmitQuiz` and `_submitQuiz`.

Reply:
It is still triggered by click .o_wslides_js_lesson_quiz_submit, should I try to not use _onSubmitQuiz in all other cases ?

Reply:
Then yes other calls not triggered by the click event should go to `_submitQuiz` directly.